test/Microsoft.AspNet.Mvc.FunctionalTests/TryValidateModelTest.cs

```diff
@@ -38,12 +38,6 @@ public class TryValidateModelTest
             // Assert
             var body = await response.Content.ReadAsStringAsync();
             var json = JsonConvert.DeserializeObject<Dictionary<string, string>>(body);
-            Assert.Equal(8, json.Count);
-            Assert.Equal("CompanyName cannot be null or empty.", json["product.CompanyName"]);
-            Assert.Equal("The field Price must be between 20 and 100.", json["product.Price"]);
-            Assert.Equal("The Category field is required.", json["product.Category"]);
-            Assert.Equal("The field Contact Us must be a string with a maximum length of 20."+
-                "The field Contact Us must match the regular expression '^[0-9]*$'.", json["product.Contact"]);
```

Comment:
Assuming from the changes that this test was incorrect. It would be good to see it verify the count of message still - as is this feature would totally regress and you'd never ever know (at least not from this test).